components/logins/src/ffi.rs

```diff
@@ -45,6 +45,7 @@ pub mod error_codes {
     pub const INVALID_LOGIN_DUPLICATE_LOGIN: i32 = 64 + 2;
     pub const INVALID_LOGIN_BOTH_TARGETS: i32 = 64 + 3;
     pub const INVALID_LOGIN_NO_TARGET: i32 = 64 + 4;
+    pub const INVALID_LOGIN_UNSPECIFIED: i32 = 64 + 5;
```

Comment:
I fear we may be about to proliferate error codes here, are we sure we want to individually expose each of these over the FFI?

Reply:
@rfk I wasn't sure how much detail needed to passed on to the caller. I was assuming the android team wanted this level of specificity but I may have misinterpreted @thomcc's [comment](https://github.com/mozilla/application-services/pull/2101#discussion_r344299048) on my first PR.

Reply:
This was done intentionally, as the lockwise team indicated that they'd like to know a precise reason the records were invalid.